docs/APIReference-Editor.md

```diff
@@ -106,6 +106,31 @@ for details on usage.
 
 ### Behavior (Optional)
 
+### autoCapitalize?: string
+
+```
+autoCapitalize?: string
+```
+
+Set if auto capitalization is turned on and how it behaves. More about platform availability and usage can [be found on mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#attr-autocapitalize).
```

Comment:
I stand 'autocorrected' ha ha :)

You're right, since these all take strings and not booleans, the wording you had makes sense. And linking to the MDN docs will hopefully keep this up to date with no changes. Nice!